src/election-param.js

```diff
@@ -1,6 +1,24 @@
+import { Config } from "@baltimorecounty/javascript-utilities";
+const { getValue, setConfig } = Config;
+const configValues = {
```

Comment:
i probably would have expected a space between lines 2 and 3, but that is not explicitly stated anywhere so not blocking.

Reply:
I updated the spacing